db/migrate/20170421154535_create_default_price_groups_for_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDefaultPriceGroupsForUsers < ActiveRecord::Migration
+
+  def up
+    User.find_each(&:create_default_price_group!)
```

Comment:
If we have users who have custom price group memberships, do we want to preserve those? i.e. instead of removing existing memberships, if they have a base or default membership, we just do nothing?